features/addressing/managing_countries/editing_country_with_provinces.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country with provinces
+    In order to add or remove provinces in existing countries
+    As an Administrator
+    I want to be able to edit a country and its provinces
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has country "United Kingdom"
+
+    @ui @javascript
+    Scenario: Adding province to existing country
+        Given I want to edit this country
```

Comment:
I do not agree, IMO this is readable to the same extent as using country's name. This feature is using in every scenarios only one country and writing `this country` is more natural for me than using in every step country's name.